server/index.js

```diff
@@ -19,10 +19,19 @@ var randomName = require('./randomname');
 
 // Create Express webapp.
 var app = express();
-var mediadevicesPath = path.join(__dirname, '../examples/mediadevices/public');
+
+// Set up the paths for the examples.
+[
+  'mediadevices',
+  'localvideosnapshot'
```

Comment:
Can you think of a way to load these dynamically? Otherwise we have to duplicate.